dev/src/pool.ts

```diff
@@ -156,5 +174,6 @@ export class ClientPool<T> {
         }
       }
     });
+    return idleClients - 1;
```

Comment:
Is this supposed to represent the number of clients deleted? Can you add a comment to clarify that?